nibabel/freesurfer/io.py

```diff
@@ -44,24 +45,52 @@ def _fread3_many(fobj, n):
     return (b1 << 16) + (b2 << 8) + b3
 
 
-def read_geometry(filepath):
+def _read_volume_info(fobj):
+    volume_info = OrderedDict()
+    head = np.fromfile(fobj, '>i4', 3)
+    if any(head != [2, 0, 20]):
+        warnings.warn("Unknown extension code.")
```

Comment:
How about `return volume_info` here so we can get rid of the indentation below?  I think that makes it easier to read, because you don't have to wonder if the code is going to do something else for this case.